cli/utils/prepare-library-package.js

```diff
@@ -2,13 +2,24 @@
 'use strict';
 
 const fs = require('fs-extra');
+const path = require('path');
 const logger = require('@blackbaud/skyux-logger');
 const skyPagesConfigUtil = require('../../config/sky-pages/sky-pages.config');
 
 function makePackageFileForDist() {
-  const contents = fs.readJsonSync(
-    skyPagesConfigUtil.spaPath('package.json')
-  );
+  const packageJsonRelativeDir = path.join('src', 'app', 'public', 'package.json');
+  const packageJsonAbsolutePath = skyPagesConfigUtil.spaPath(packageJsonRelativeDir);
+
+  if (!fs.existsSync(packageJsonAbsolutePath)) {
```

Comment:
This is a breaking change for libraries, requiring them to provide their own package.json file at **./src/app/public**.